### DISCUSSION TITLE
How to open 2 URLs in 2 different tabs at a same time during the exam using "Allow address bar for additional windows" option in User Interface settings?

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#602] [Q&A]
Hi Team,

We need to know that "How to open 2 URLs in 2 different tabs at a same time during the exam using "Allow address bar for additional windows" option in User Interface settings?"
We have enabled the both "Allow address bar for additional windows" option and "Use browser window" option in User Interface settings.
But, While launching the exam only the Start URL which we have given was opened. We could not able to see the "new tab" option in SEB.
So, Is there any other option available to open 2 tabs in SEB or do we need to configure this option by developing code/script?

Kindly assist on this.

**Comment by dbuechel** [comment]
There is currently no way to open multiple start URLs at once. However, you certainly can open an additional window using e.g. JavaScript as you already correctly stated. You could also make a link open in an additional window by setting its HTML target attribute respectively.

**Reply by Chris251290** [reply]
Hi Damian,

Thanks for your respose.

Yes. As you stated, There is currently no way to open multiple start URLs at once.
If I allow any URL link in URL filtering, I can able to access the URL while attending the exam If I gave that URL in Exam Section. That URL will open like a window. 

Is there any option like to click and open "New Tab" in SEB like Google Chrome and other browsers?

**Reply by dbuechel** [reply]
No, at the moment, SEB does not yet support browser tabs. We do have that feature in our backlog, but it is unlikely that we'll be able to implement it this year.
